config/locales/en.yml

```diff
@@ -1,5 +1,7 @@
 ---
 en:
+  automated_tests:
+    next_token_generation: Next Token Generation Time
```

Comment:
1. Only capitalize the first word (to make it consistent with the other labels on the autotesting page)
2. Move this string to a more specific locale file (see `automated_testing/views`). We're slowly moving strings out of this general file to better categorize them.